test/unit/test_hub_config.py

```diff
@@ -128,3 +147,55 @@ def test_hub_missing_tasks_config(hubverse_hub):
     with pytest.raises(FileNotFoundError):
         hc = HubConfig(hub_path, tasks_file="missing-tasks.json")
         print(hc)
+
+
+def test_get_task_id_values_all_rounds(hubverse_hub):
+    hub_path = AnyPath(hubverse_hub)
+    hc = HubConfig(hub_path)
+
+    assert hc.get_task_id_values() == {
+        "reference_date": {"2024-07-13", "2024-07-21", "2024-08-13"},
+        "target": {"borg growth rate change", "wk number assimilations", "starfleet entrance exam score"},
+        "horizon": {-1, 0, 1, 2, 3, "one", "two", "three"},
+        "location": {"Earth", "Vulcan", "789", "Ryza", "123", "Bajor"},
+        "target_end_date": {"2024-07-20", "2024-07-27", "1999-12-31", "2024-08-20"},
+        "age": {10, 20, 30, 40, 50},
+    }
+
+
+def test_get_task_id_values_single_round(hubverse_hub):
+    hub_path = AnyPath(hubverse_hub)
+    hc = HubConfig(hub_path)
+
+    assert hc.get_task_id_values("Round 2") == {
+        "target": {"starfleet entrance exam score"},
+        "age": {10, 20, 30, 40, 50},
+    }
+
+
+def test_get_task_id_values_missing_round(hubverse_hub):
+    hub_path = AnyPath(hubverse_hub)
+    hc = HubConfig(hub_path)
+
+    with pytest.raises(ValueError):
+        hc.get_task_id_values("missing round") == {
+            "target": {"starfleet entrance exam score"},
+            "age": {10, 20, 30, 40, 50},
+        }
```

Comment:
Is the equality comparison here a non sequitur?
```suggestion
        hc.get_task_id_values("missing round")
```

Reply:
Yes--good call!